tests/rest/client/test_relations.py

```diff
@@ -1130,17 +1126,21 @@ def _send_relation(
         if not access_token:
             access_token = self.user_token
 
-        query = ""
-        if key:
-            query = "?key=" + urllib.parse.quote_plus(key.encode("utf-8"))
-
         original_id = parent_id if parent_id else self.parent_id
 
+        if content is None:
+            content = {}
+        content["m.relates_to"] = {
+            "event_id": original_id,
+            "rel_type": relation_type,
+        }
+        if key is not None:
+            content["m.relates_to"]["key"] = key
```

Comment:
This pretty much inlines the logic from `/send_relation` into the helper method for tests.